rotkehlchen/api/v1/schemas.py

```diff
@@ -2567,14 +2570,76 @@ def transform_data(
             raise ValidationError(f'Could not deserialize data: {e!s}') from e
 
 
-class LocationAssetMappingsUpdateSchema(LocationAssetMappingsBaseSchema):
+class LocationAssetMappingsUpdateSchema(Schema):
```

Comment:
why this change? I don't see why you need to change this

Reply:
oh the superclass used here is wrong. the reason an error wasn't thrown is because the field has `load_default=None`